examples/tutorial/src/election.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Election,
+  ElectionStatus,
+  PlainCensus,
+  UnpublishedElection,
+  VocdoniSDKClient
+} from '@vocdoni/sdk'
+
+export const createElection = (census: PlainCensus): UnpublishedElection => {
+  const election: UnpublishedElection = Election.from({
+    title: 'Election title',
+    description: 'Election description',
+    header: 'https://source.unsplash.com/random',
+    endDate: new Date().getTime() + 100000,
+    census,
+    electionType: {
+      // This is the default behavior
+      secretUntilTheEnd: false
```

Comment:
`secretUntilTheEnd` is `false` by default, so I wouldn't pass the `electionType` here to make it even easier.